lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
I see your Calendar module is also used as your documentation entry-point but that is easy to customize on ExDoc. :)